web/src/components/vis/CorrelationTile.vue

```diff
@@ -73,6 +77,32 @@ export default {
             value: Math.abs(d.value),
           }));
     },
+    searchResults() {
+      if (!this.currentUserInput || this.currentUserInput.length === 0) {
+        return [];
+      }
+      return this.searchBarResults;
```

Comment:
I would probably make `currentUserInput` default to `''`, so you don't have to think about more than 1 data type.  It's my understanding that `null` and `''` carry the same meaning here: search box empty

Then, you can either check truthiness or you can use strict equality `if (this.currentUserInput === '') {...`

I think strict equality is nicer personally.